### PR TITLE
[NG] fix: clDgRowSelection attribute naming (#2019)

### PR DESCRIPTION
**Pull request by jewway**
- Fix data grid row selection attribute typo.

- I can't run **npm install** , then i use yarn to install but run **npm test** fail , package missed `@ngtools/webpack `

**NPM version : 5.6.0**

image

Signed-off-by: Jewway jewway@gmail.com

### CODE REVIEW

File: src/clr-angular/data/datagrid/datagrid.ts [hunk 1]
```diff
@@ -124,7 +124,7 @@ export class ClrDatagrid implements AfterContentInit, AfterViewInit, OnDestroy {
     /**
      * Selection/Deselection on row click mode
      */
-    @Input("clDgRowSelection")
+    @Input("clrDgRowSelection")
```
Comment (hippee-lee) [inline src/clr-angular/data/datagrid/datagrid.ts]: I think we need to stay backwards compatible here. I believe that you can keep the new input and re-add the original ```@Input("clDgRowSelection")```

Comment (jewway) [reply src/clr-angular/data/datagrid/datagrid.ts]: @hippee-lee 
You're right. I've considered it in the implementation. I'll add it again.

Thank you for your reminder.

Comment (jewway) [reply src/clr-angular/data/datagrid/datagrid.ts]: @hippee-lee re-added, I am not sure whether the Implementation meets your coding rule.

File: src/clr-angular/data/datagrid/datagrid.ts [hunk 2]
```diff
@@ -124,11 +124,20 @@ export class ClrDatagrid implements AfterContentInit, AfterViewInit, OnDestroy {
...
         this.selection.rowSelectionMode = value;
     }
 
+    /**
+     * stay backwards compatible , will be renamed to clrDgRowSelection
+     * @deprecated
+     */
+    @Input("clDgRowSelection")
+    set rowSelectionModeDeprecated(value: boolean) {
+        this.selection.rowSelectionMode = value;
```
[the remark this answers is not in the document]
Comment (jewway) [reply src/clr-angular/data/datagrid/datagrid.ts]: I agree with you, i will create a setter method to handle same logic.

**Comment by hippee-lee** [comment]
Hi @jewway, 
I'd like to add you to the list of Clarity contributors on the website. If you would like to be added please let me know your full name. 
Thanks.

**Comment by jewway** [comment]
Hi @hippee-lee ,
Thank you so much for inviting me ,  this is my pleasure.
I really like this project , if there's anything I can help,don't hesitate to tell me.
My full name is **Wei Cheng Chu**

**Comment by jewway** [comment]
@hippee-lee - Thank you !

**Pull request merged**